tests/test_graphql.py

```diff
@@ -66,10 +65,6 @@ async def test_query_environment_project(server, client, setup_database):
               id
               halted
               isExpertMode
-              project
```

Comment:
Why remove this? We could still query for it by adding the `project_default` fixture to this test, right ?

Reply:
I removed it because the goal of the test was no longer to test the relationship to project, but it was to test `isExpertMode`. I renamed the test but added `project_default` back anyway